MyBooksDlg.obj : error LNK2019: _mysql_init@4"public: void __thiscall CMyBooksDlg::OnBnClickedBtnload(void)" (?OnBnClickedBtnload@CMyBooksDlg@@QAEXXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
MyBooksDlg.obj : error LNK2019: _mysql_real_connect@32"public: void __thiscall CMyBooksDlg::OnBnClickedBtnload(void)" (?OnBnClickedBtnload@CMyBooksDlg@@QAEXXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
MyBooksDlg.obj : error LNK2019: _mysql_query@8"public: void __thiscall CMyBooksDlg::OnBnClickedBtnload(void)" (?OnBnClickedBtnload@CMyBooksDlg@@QAEXXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
MyBooksDlg.obj : error LNK2019: _mysql_store_result@4"public: void __thiscall CMyBooksDlg::OnBnClickedBtnload(void)" (?OnBnClickedBtnload@CMyBooksDlg@@QAEXXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
MyBooksDlg.obj : error LNK2019: _mysql_free_result@4"public: void __thiscall CMyBooksDlg::OnBnClickedBtnload(void)" (?OnBnClickedBtnload@CMyBooksDlg@@QAEXXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
MyBooksDlg.obj : error LNK2019: _mysql_fetch_row@4"public: void __thiscall CMyBooksDlg::OnBnClickedBtnload(void)" (?OnBnClickedBtnload@CMyBooksDlg@@QAEXXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
MyBooksDlg.obj : error LNK2019: _mysql_close@4"public: void __thiscall CMyBooksDlg::OnBnClickedBtnload(void)" (?OnBnClickedBtnload@CMyBooksDlg@@QAEXXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
C:\Program Files\MySQL\MySQL Server 8.0\lib\libmysql.lib : warning LNK4272: 'x64' 라이브러리 컴퓨터 종류가 'x86' 대상 컴퓨터 종류와 충돌합니다.
C:\MyBooks\MyBooks\Debug\MyBooks.exe : fatal error LNK1120: 7개의 확인할 수 없는 외부 참조입니다.
